examples/docker-helloworld.yaml

```diff
@@ -30,6 +30,7 @@ gateway:
     - controller
     - registry
   container_name: gateway
+  command: /usr/bin/run_filebeat.sh
```

Comment:
Also, for helloworld, we need to use the env section and define the A8_SERVICE variable as the python code depends on it.